tensorflow/core/kernels/data/map_dataset_op_test.cc

```diff
@@ -0,0 +1,363 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <tuple>
+#include "tensorflow/core/framework/dataset.h"
```

Comment:
add an empty line here (after the `<tuple>` line)

Reply:
As the named struct is used to replace the tuple for the test parameters, this include is not required now. I will remove it.